### PR TITLE
masterSource

### PR DESCRIPTION
**Pull request by imtithalhalalo**
Moving master source from FormShell props to be sent to the action button Integration Action
please review the pages that has change in master source

### CODE REVIEW

**Comment by omarZrayka23** [comment]
Screenshot 2024-11-27 at 3 05 51 PM


The address in the textArea field has to be verified in the window

**Comment by omarZrayka23** [comment]
Screenshot 2024-11-27 at 3 21 41 PM

File: src/pages/fi-accounts/forms/AccountsForm.js
Comment (RebeccaFrem02) [inline src/pages/fi-accounts/forms/AccountsForm.js:1]: image
the clear button of the lookup is not working

Comment (RebeccaFrem02) [reply src/pages/fi-accounts/forms/AccountsForm.js:1]: check the onClear function

Comment (RebeccaFrem02) [reply src/pages/fi-accounts/forms/AccountsForm.js:1]: image
check the functionality of the clear function of this field

**Review by omarZrayka23 — APPROVED** [review]

**Pull request merged**